openedx/core/djangoapps/user_api/views.py

```diff
@@ -158,9 +158,10 @@ def post(self, request):
 class RegistrationView(APIView):
     """HTTP end-points for creating a new user. """
 
-    DEFAULT_FIELDS = ["email", "name", "username", "password"]
+    DEFAULT_FIELDS = ["email", "username", "password"]
```

Comment:
set `REGISTRATION_EXTRA_FIELDS` setting to empty list and create a new Form via `REGISTRATION_EXTENSION_FORM` having those additional fields as described [here](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/customize_registration_page.html)